ledger/simulation/simulator.go

```diff
@@ -47,21 +43,50 @@ type Request struct {
 	TraceConfig           ExecTraceConfig
 }
 
-// Latest is part of the LedgerForSimulator interface.
+// simulatorLedger patches the ledger interface to use a constant latest round.
+type simulatorLedger struct {
+	*data.Ledger
+	start basics.Round
+}
+
+// Latest is part of the ledger.Ledger interface.
 // We override this to use the set latest to prevent racing with the network
-func (l simulatorLedger) Latest() basics.Round {
+func (l *simulatorLedger) Latest() basics.Round {
 	return l.start
 }
 
+// LatestTotals is part of the ledger.Ledger interface.
+func (l *simulatorLedger) LatestTotals() (basics.Round, ledgercore.AccountTotals, error) {
+	totals, err := l.Totals(l.start)
+	return l.start, totals, err
+}
+
 // LookupLatest would implicitly use the latest round in the _underlying_
 // Ledger, it would give wrong results if that ledger has moved forward. But it
 // should never be called, as the REST API is the only code using this function,
 // and the REST API should never have access to a simulatorLedger.
-func (l simulatorLedger) LookupLatest(addr basics.Address) (basics.AccountData, basics.Round, basics.MicroAlgos, error) {
+func (l *simulatorLedger) LookupLatest(addr basics.Address) (basics.AccountData, basics.Round, basics.MicroAlgos, error) {
```

Comment:
Good point. I originally changed this when debugging because I thought it was causing an issue, but I just reverted it locally and tests still pass, so I will change it back